docs/userguide.md

```diff
@@ -93,6 +93,9 @@ automated, then invoke the deployment script by running:
 ./scripts/deploy-federation-latest.sh cluster2
 ```
 
+The above script will join the host cluster to federation control panel automatically.
```

Comment:
s/panel/plane/

Reply:
Can be modified to:
The above script joins the host cluster to the federation control plane it deploys, by default.

Reply:
Done